_proto/hapi/services/tiller.proto

```diff
@@ -304,3 +309,17 @@ message GetHistoryRequest {
 message GetHistoryResponse {
 	repeated hapi.release.Release releases = 1;
 }
+
+// TestReleaseRequest is a request to get the status of a release.
+message TestReleaseRequest {
+	// Name is the name of the release
+	string name = 1;
+	// timeout specifies the max amount of time any kubernetes client command can run.
+	int64 timeout = 2;
+}
+
+// TestReleaseResponse
+message TestReleaseResponse {
+	// TODO: change to repeated hapi.release.Release.Test results = 1; (for stream)
```

Comment:
Not tracking TODOs anywhere. They're usually just for me. I don't push them into master but I just forgot this one.